src/main/java/address/controller/PersonOverviewController.java

```diff
@@ -129,6 +129,15 @@ private void handleDeletePersons() {
         }
     }
 
+    private boolean isListContainingNullObject(List<ReadOnlyViewablePerson> selected) {
+        for (Object obj : selected) {
+            if (obj == null){
+                return true;
+            }
+        }
+        return false;
+    }
```

Comment:
```java
public static boolean collectionContainsNull(Collection<?> col) {
    return col.stream().anyMatch(Objects::isNull);
}
```
Not sure why argument parameter has to be ReadOnlyViewablePerson, and can use stream to make things neater